api/catalog/api/views/image_views.py

```diff
@@ -111,6 +112,11 @@ def thumbnail(self, request, *_, **__):
         if not image_url:
             raise get_api_exception("Could not find image.", 404)
 
+        # Hotfix to use scaled down version of the image from SMK
+        if "iip.smk.dk" in image_url:
```

Comment:
Could this potentially also use a regex `search`? I believe it'd be much faster which seems preferred given that this will be run on every thumbnail URL.

Reply:
Wow! That takes me totally by surprise, did a little testing and this proves it even more IMO:

```python
[ins] In [1]: import re

[ins] In [2]: urls = [
         ...:     "https://live.staticflickr.com/7042/6962402467_5434c54897_b.jpg",
         ...:     "https://live.staticflickr.com/7067/6816284396_1c4e5d8ffd_b.jpg",
         ...:     "https://live.staticflickr.com/7048/6962402757_e47732eb67_b.jpg",
         ...:     "https://live.staticflickr.com/7053/6816284264_af07bae1ae.jpg  ",
         ...:     "https://live.staticflickr.com/7070/6962402365_273ee6de73_b.jpg",
         ...:     "https://live.staticflickr.com/7197/6816283612_818cec588e_b.jpg",
         ...:     "https://live.staticflickr.com/7206/6816283716_d3dfd707d5_b.jpg",
         ...:     "https://live.staticflickr.com/7057/6816283724_dd723fd8e0_b.jpg",
         ...:     "https://live.staticflickr.com/7185/6816283592_d49a5f0739_b.jpg",
         ...:     "https://live.staticflickr.com/7040/6962401837_137a183b53_b.jpg",
         ...:     "https://live.staticflickr.com/7206/6816282540_da3b3db410_b.jpg",
         ...:     "https://live.staticflickr.com/7069/6962401633_c39f008a81_b.jpg",
         ...:     "https://live.staticflickr.com/7036/6816283028_5776c046b2_b.jpg",
         ...:     "https://iip.smk.dk/iiif/jp2/KKSgb5100_34.TIF.jp2/full/!2048,/0/default.jpg",
         ...:     "https://live.staticflickr.com/7062/6816283340_0d75fcde2d_b.jpg",
         ...:     "https://live.staticflickr.com/7179/6816283290_b143cc5112.jpg  ",
         ...:     "https://live.staticflickr.com/7043/6962401447_ed46d7568f_b.jpg",
         ...:     "https://live.staticflickr.com/7067/6962401181_70ce4835e0_b.jpg",
         ...:     "https://live.staticflickr.com/7055/6816282762_df50695285_b.jpg",
         ...:     "https://live.staticflickr.com/7041/6962400979_a3f00f5beb_b.jpg",
         ...:     "https://live.staticflickr.com/7052/6816282622_d2ce5f25c3_b.jpg",
         ...:     ]
         ...: 

[ins] In [3]: %timeit ["iip.smk.dk" in url for url in urls]
776 ns ± 11 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

[ins] In [4]: match_pat = re.compile(".*?iip\.smk\.dk.*")

[ins] In [5]: %timeit [match_pat.match(url) for url in urls]
15.1 µs ± 87.3 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

[ins] In [6]: %timeit [match_pat.search(url) for url in urls]
354 µs ± 1.55 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)

[ins] In [7]: anchor_pat = re.compile("^https?://iip\.smk\.dk.*")

[ins] In [8]: %timeit [anchor_pat.match(url) for url in urls]
3.7 µs ± 13.9 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

[ins] In [9]: %timeit [anchor_pat.search(url) for url in urls]
9.86 µs ± 328 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```

`in` it is 😄